core/server/fluxruntime_test.go

```diff
@@ -201,6 +204,30 @@ func TestListFluxRuntimeObjects(t *testing.T) {
 	g.Expect(res.Deployments[0].Name).To(Equal(fluxDep.Name))
 }
 
+func TestListFluxRuntimeObjectsResponseError(t *testing.T) {
```

Comment:
@luizbafilho Do you see an easy way to make this test pass when running all tests at the same time? This only passes if it runs individually 😔

Reply:
NVM I think I fixed this